x-pack/legacy/plugins/reporting/index.js

```diff
@@ -92,6 +92,11 @@ export const reporting = (kibana) => {
           timeout: Joi.number().integer().default(120000),
         }).default(),
         capture: Joi.object({
+          networkPolicy: Joi.object({
+            enabled: Joi.boolean().default(true),
+            allow: Joi.array().items(Joi.string()).default(['http:', 'https:', 'ws:', 'wss:', 'data:']),
+            deny: Joi.array().items(Joi.string()).default([]),
+          }).default(),
```

Comment:
Source of new config is here, allows for setting allow/deny lists. Implementation below